tensorboard/plugins/scalar/summary_v2.py

```diff
@@ -30,6 +30,34 @@
 def scalar(name, data, step=None, description=None):
     """Write a scalar summary.
 
+    See also `tf.summary.image`, `tf.summary.histogram`, `tf.summary.SummaryWriter`.
+
+    Logs simple numeric values for later analysis in TensorBoard.  Each summary
+    point is associated with an integral `step` value. This enables the incremental
+    logging of time series data.  A common usage of this API is to log loss during
+    training to produce a loss curve.
+
+    For example:
+
+    ```python
+    test_summary_writer = tf.summary.create_file_writer('test/demo_log_dir')
+    with test_summary_writer.as_default():
+        tf.summary.scalar('loss', 0.345, step=1)
+        tf.summary.scalar('loss', 0.234, step=2)
+        tf.summary.scalar('loss', 0.123, step=3)
+    ```
+
+    Multiple independent time series may be logged by giving each series a unique
```

Comment:
This paragraph and the "There is no constraint..." paragraph below on L56 apply to all the tf.summary.* APIs, not just scalars.  I wonder if it makes more sense to move these 2 paragraphs out of here, and maybe more appropriate in the "overview" page for the entire tf.summary module?  (https://www.tensorflow.org/api_docs/python/tf/summary)

Reply:
I suspect, as the most fundamental type, that the `tf.scalar` symbol will be most users' first interaction with the summary API.  I support adding this information to in the higher`tf/summary` documentation, but think it is valuable here also.

Reply:
Sounds reasonable to me.

Reply:
I agree that we probably do want some general docs on the module about common behavior.  Maybe after the individual symbol updates we can take a pass over the entire module to harmonize the docstrings a little and link to common module-level docs as need be.